package.json

```diff
@@ -5,6 +5,7 @@
     "main": "index.js",
     "scripts": {
         "start": "node index.js",
+        "start-dev": "webpack -w --mode=development --env.NODE_ENV=development & nodemon index.js --env.NODE_ENV=development",
```

Comment:
Is this meant to be `&` or `&&`?

Reply:
So using `&` seems to hang at wepack. I think this is maybe because both commands don't complete/stop running unless something is wrong?

Reply:
As long as the intention is for webpack to run in the background, `&` is fine.